packages/clerk-js/src/core/auth/cookies/clientUat.ts

```diff
@@ -0,0 +1,51 @@
+import { createCookieHandler } from '@clerk/shared/cookie';
+import { addYears } from '@clerk/shared/date';
+import type { ClientResource } from '@clerk/types';
+
+import { inCrossOriginIframe } from '../../../utils';
+import { getCookieDomain } from '../getCookieDomain';
+
+const CLIENT_UAT_COOKIE_NAME = '__client_uat';
+
+export type ClientUatCookieHandler = {
+  set: (client: ClientResource | undefined) => void;
+  get: () => number;
+};
+
+export const createClientUatCookie = (): ClientUatCookieHandler => {
+  const clientUatCookie = createCookieHandler(CLIENT_UAT_COOKIE_NAME);
```

Comment:
There's a comment in `session.ts` that mentions what the cookie is used for, maybe we can do the same here and in `devBrowser`?

Reply:
https://github.com/clerk/javascript/pull/3362#issuecomment-2110029255